doc/rofi.1.markdown

```diff
@@ -36,16 +36,16 @@ To show the run dialog:
 
 **rofi** can emulate **dmenu(1)** (a dynamic menu for X) when launched with the `-dmenu` flag.
 
-The "official" website for `dmenu` can be found [here](http://tools.suckless.org/dmenu/).
+The official website for `dmenu` can be found [here](http://tools.suckless.org/dmenu/).
```

Comment:
@DaveDavenport Is it official or not? If so, then dropping the quotes If not really official, then "unofficial" ;)